server/routes/boards.test.js

```diff
@@ -0,0 +1,18 @@
+import http from 'http';
+import server from './boards';
+import { PORT } from '../configs/config';
+
+describe('httpServer', () => {
```

Comment:
how are you starting the server? Check [this](https://github.com/visionmedia/supertest) for alternate option.

Reply:
Done !!